CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to the "vscode-debug-adapter-apache-camel" extension will be
 ## 0.7.0
 
 - Provide Codelens to `Camel Run with JBang`
+- Provide `Run with JBang` with automatic reload `--dev`
```

Comment:
I think it will be useful for end-users to have a less technical description. Something like: `Run with JBang` command are now using automatic reloading. It corresponds to `--dev` option.

and with a link to the Camel JBang dev documentation.

What do you think?

Reply:
You are right, it will be better to understand for users.

Updated